static/js/components/EducationTab.js

```diff
@@ -97,48 +99,70 @@ class EducationTab extends ProfileTab {
     updateProfile(clone);
   }
 
-  printAddDegree(level){
-    const { ui: { educationDegreeInclusions } } = this.props;
-    if (educationDegreeInclusions[level.value]){
-      let openForm = () => this.openNewEducationForm(level.value, null);
-      return <Grid key={`add-${level.value}`}>
-        <Cell col={11}></Cell>
-        <Cell col={1}>
-          <FABButton mini onClick={openForm} raised ripple>
-            <Icon name="add" />
-          </FABButton>
-        </Cell>
-      </Grid>;
+  renderEducationLevel(level){
+    const { ui: { educationDegreeInclusions }, profile: { education } } = this.props;
+    if (educationDegreeInclusions[level.value]) {
+      let rows = [];
+      if (education !== undefined) {
+        rows = education.filter(education => education.degree_name === level.value).
+          map((entry, i) => this.educationRow(entry, i));
+      }
+      let fabSpacing = () => {
+        let len = _.isArray(education) ? education.length : 0;
+        switch ( len ) {
+        case 0:
+          return "no-entries";
+        case 1:
+          return "one-entry";
+        default:
+          return "more-entries";
+        }
+      };
+      rows.push(
+        <FABButton
+          colored
+          onClick={() => this.openNewEducationForm(level.value, null)}
+          key="I'm unique!"
+          className={`profile-add-button education-add-button ${fabSpacing()}`}>
+          <Icon name="add" />
+        </FABButton>
+      );
+      return rows;
     }
```

Comment:
this is so similar to the function on `EmploymentTab`...probably not worth abstracting it right now but maybe something to do later.